frontend/apps/filemanager/filemanager.lua

```diff
@@ -41,6 +42,20 @@ local function restoreScreenMode()
     end
 end
 
+local function truncatePath(text)
+    local screen_width = Screen:getWidth()
+    local face = Font:getFace("xx_smallinfofont")
+    local reserve_text = string.reverse(text)
```

Comment:
Oh there's a little typo in this variable name. Probably best to name it `reversed_text` :-)

Reply:
Of course :-)